package.json

```diff
@@ -53,13 +53,17 @@
   },
   "scripts": {
     "check-license": "./scripts/check-license.sh",
-    "check-ls": "npm ls --loglevel=http --parseable 1>/dev/null && echo '# npm is in a good state'",
-    "cover": "babel-node ./node_modules/.bin/babel-istanbul cover ./node_modules/.bin/_mocha -- test/ test/samplers/ test/baggage/",
+    "check-ls":
+      "npm ls --loglevel=http --parseable 1>/dev/null && echo '# npm is in a good state'",
+    "cover":
+      "babel-node ./node_modules/.bin/babel-istanbul cover ./node_modules/.bin/_mocha -- test/ test/samplers/ test/baggage/",
     "coveralls": "npm run cover && cat ./coverage/lcov.info | coveralls",
     "flow": "flow",
-    "lint": "eslint $(ls src/ | grep '.js$') && echo '# linter passed' && npm run check-license",
+    "lint":
+      "eslint $(ls src/ | grep '.js$') && echo '# linter passed' && npm run check-license",
     "test": "make test",
-    "test-all": "npm run test-core && npm run test-samplers && npm run test-crossdock && npm run test-baggage",
+    "test-all":
+      "npm run test-core && npm run test-samplers && npm run test-crossdock && npm run test-baggage",
```

Comment:
can we do without this pure formatting change?

Reply:
It does seem superfluous. However, the formatting changes are being done automatically by [Prettier](https://github.com/prettier/prettier), which is fantastic and should arguably be adopted on all JS projects. Up to now, I've just been toggling the package to disabled and then re-enabled in my editor. But I've been thinking we should consider [using Prettier](https://prettier.io/) in this project. [A number of 'big name' projects use it,](https://github.com/prettier/prettier/issues/1351) including Babel, and projects at Uber :)

Reply:
I don't have objection to auto-formatting, I think we also use it in jaeger-ui.

But in that case we need to introduce it as a separate PR which reformats all existing source code without making any other changes. It's not a good idea to mix formatting changes with logic changes.

Reply:
Agreed, completely. (It was a mistake.) I'll make a PR for prettier.